### PR TITLE
feat: use eslint-config-screwdriver@5

### PR DESCRIPTION
**Pull request by jithine**
## Context

Upgrade to eslint-config-screwdriver@5 which brings new eslint version and rules.

## References

https://github.com/screwdriver-cd/eslint-config-screwdriver/pull/40

## License
I confirm that this contribution is made under a BSD license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by jithine** [comment]
cc @tkyi @screwdriver-cd/yahoo-japan 

PR will fail until https://github.com/screwdriver-cd/eslint-config-screwdriver/pull/40 is merged

File: plugins/templates/listVersionsWithMetric.js [hunk 1]
```diff
@@ -9,8 +9,10 @@ module.exports = () => ({
     method: 'GET',
     path: '/templates/{name}/metrics',
     config: {
-        description: 'Get all template versions and metrics for a template name with pagination',
-        notes: 'Returns all template records and associated metrics for a given template name',
+        description:
```
Comment (tkyi) [inline plugins/templates/listVersionsWithMetric.js]: I don't know if I love this change.
So many unnecessary newlines

File: plugins/commands/updateTrusted.js [hunk 2]
```diff
@@ -35,14 +35,19 @@ module.exports = () => ({
             });
 
             if (commands.length === 0) {
-                throw boom.notFound(`Command ${namespace}/${name} does not exist`);
+                throw boom.notFound(
+                    `Command ${namespace}/${name} does not exist`
```
Comment (tkyi) [inline plugins/commands/updateTrusted.js]: idk if this rule makes sense to me

Comment (tkyi) [reply plugins/commands/updateTrusted.js]: there's a lot of extra newlines

Comment (jithine) [reply plugins/commands/updateTrusted.js]: should make it consistent https://github.com/screwdriver-cd/eslint-config-screwdriver/pull/41/files

Comment (jithine) [reply plugins/commands/updateTrusted.js]: I will update this PR once that rule goes int

**Comment by jithine** [comment]
closing in favor of #2023

**Pull request closed without merging**